### DISCUSSION TITLE
Fundamentos da orientação a objetos

### DISCUSSION THREAD

**Discussion by LuizH1359** [balta-io/forum#185] [Tira Dúvidas] [answered]
Primeiramente, estou amando o curso cada vez mais apaixonado !
Porém quando fui colocar a mão na massa notei que quando declaro um prop e vou preencher aparece um erro dizendo que o valor da
propriedade precisa sair não nulo do construtor, até procurei "resolver" o problema com uma sugestão do vscode porém acredito que isso não seja saudável, poderia me ajudar a entender o porque de estar acontecendo somente na minha máquina, e como resolver também? 
![duvida](https://user-images.githubusercontent.com/94483864/209465666-ede3f44e-f77f-40ff-8253-f5b7d016de5b.jpg)

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @LuizH1359, como vai?

Muito obrigado pelo feedback, é ótimo saber que estamos no caminho certo 💜

Este vídeo aqui explica como mitigar seu problema:
[15:23](https://www.youtube.com/watch?v=PNasRBbo00g&t=923s) - Nullables

**Comment by LuizH1359** [comment]
Em dom., 25 de dez. de 2022 às 21:36, Andre Baltieri <
***@***.***> escreveu:

> Boa noite, @LuizH1359 <https://github.com/LuizH1359>, como vai?
>
> Muito obrigado pelo feedback, é ótimo saber que estamos no caminho certo
> 💜
>
> Este vídeo aqui explica como mitigar seu problema:
> 15:23 <https://www.youtube.com/watch?v=PNasRBbo00g&t=923s> - Nullables
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/balta-io/forum/discussions/185#discussioncomment-4495043>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWQ3LGDVC46BIVU25ZWWBGTWPDSCDANCNFSM6AAAAAATI52UCQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
Obrigado pela atenção , principalmente por ser natal abraço 🤗

**Comment by LuizH1359** [comment]
Em dom., 25 de dez. de 2022 às 21:42, Luiz Henrqique Campos Rocha <
***@***.***> escreveu:

>
>
> Em dom., 25 de dez. de 2022 às 21:36, Andre Baltieri <
> ***@***.***> escreveu:
>
>> Boa noite, @LuizH1359 <https://github.com/LuizH1359>, como vai?
>>
>> Muito obrigado pelo feedback, é ótimo saber que estamos no caminho certo
>> 💜
>>
>> Este vídeo aqui explica como mitigar seu problema:
>> 15:23 <https://www.youtube.com/watch?v=PNasRBbo00g&t=923s> - Nullables
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/balta-io/forum/discussions/185#discussioncomment-4495043>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AWQ3LGDVC46BIVU25ZWWBGTWPDSCDANCNFSM6AAAAAATI52UCQ>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
> Obrigado pela atenção , principalmente por ser natal abraço 🤗

Tenha um ótimo natal , sucesso 🚀
